docs/starters.yml

```diff
@@ -6959,3 +6959,18 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://tsparticles.github.io/gatsby-landing-page-starter/
+  repo: https://github.com/tsparticles/gatsby-landing-page-starter
+  description: tsParticles Simple Landing Page Starter for Gatsby Websites
+  tags:
```

Comment:
Refer to [categories.yml](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) to see the valid tags that can be added here. You cannot have tags `Particles.js`, `tsParticles` etc. as they aren't part of `categories.yml` if you believe they are missing you can add request additions to the file. However, we recommend using the available categories.